src/index.ts

```diff
@@ -239,6 +240,25 @@ task("starknet-compile", "Compiles Starknet contracts")
     .addFlag("disableHintValidation", "Allows compiling a contract with any python code in hints.")
     .setAction(starknetCompileAction);
 
+task("starknet-compile-cairo1", "Compiles Starknet cairo1 contracts")
+    .addOptionalVariadicPositionalParam(
+        "paths",
+        "The paths to be used for deployment.\n" +
+            "Each of the provided paths is recursively looked into while searching for compilation artifacts.\n" +
+            "If no paths are provided, the default contracts directory is traversed."
+    )
+    .addOptionalParam(
+        "manifestPath",
+        "Allows to specify locally installed cairo1 compiler path.\n" +
+            "e.g. --manifest-path 'path/to/Cargo.toml' or can also be set on hardhat.config.ts file."
+    )
+    .addOptionalParam(
```

Comment:
Does cairo1 compilation use the notion of cairo-paths (paths related to importing)?

Reply:
https://github.com/Shard-Labs/starknet-hardhat-plugin/pull/328/commits/e5836b358f557103a39b64cc449cdd59404125b9

Reply:
Setting `cairoPath: ""` doesn't look good IMO (are we doing that anywhere else?). I think that's also related to having a separate options object for cairo1 compilation.